### ISSUE TITLE
Remove alpha tag from Story Promo

### ISSUE THREAD

**Issue by DenisHdz** [BBC-archive/psammead#2538]
**Is your feature request related to a problem? Please describe.**
The Story Promo component is currently an alpha component.

This component should be amended and tested according to the a11y requirements to remove the tag.

**Describe the solution you'd like**
Remove alpha tag after successful a11y swarm on the component.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by DenisHdz** [comment]
Blocked on https://github.com/bbc/simorgh/issues/4501

**Comment by DenisHdz** [comment]
Unblocked after a successful a11y swarm.

**Comment by DenisHdz** [comment]
Closed as done https://github.com/bbc/psammead/pull/3154
